### PR TITLE
[.NET] Added FormRecognizer 3.0.0 to 2020/09 release notes

### PR DESCRIPTION
**Pull request by kinelski**
Including FormRecognizer 3.0.0 out-of-band release.

### CODE REVIEW

File: releases/2020-09/dotnet.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The Azure SDK team is pleased to announce our September 2020 client library rele
 
 #### GA
 
-- _Add packages_
+- Form Recognizer
```
Comment (jongio) [inline releases/2020-09/dotnet.md:13]: Please also add as a highlight in index.md.

**Comment by jongio** [comment]
Are you going to do other langs as well? If not, who is?

File: releases/2020-09/dotnet.md [hunk 2]
```diff
@@ -36,9 +36,13 @@ If you have a bug or feature request for one of the libraries, please [file an i
 
 Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here are some of the highlights:
 
-### _Package name_ 
+### Form Recognizer [Changelog](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md#300-2020-08-20)
+
+- First stable release of the Azure.AI.FormRecognizer package.
```
Comment (maririos) [inline releases/2020-09/dotnet.md:41]: should we add the service version it targets? or it is not neccesary

File: releases/2020-09/dotnet.md (hunk 1 above)
Comment (samvaity) [reply releases/2020-09/dotnet.md:13]: Added [here](https://github.com/Azure/azure-sdk/pull/1723/files#diff-850663d875f3d8fcd42df0950e712dd7R14)

Comment (kinelski) [reply releases/2020-09/dotnet.md:13]: Done in https://github.com/Azure/azure-sdk/pull/1723

**Comment by kinelski** [comment]
> 
> 
> Are you going to do other langs as well? If not, who is?

Python has already been merged. Java has a PR out (https://github.com/Azure/azure-sdk/pull/1723) and @willmtemple is responsible for JS.

File: releases/2020-09/dotnet.md (hunk 2 above)
Comment (kinelski) [reply releases/2020-09/dotnet.md:41]: I think it's worth adding it. I'll update it.

**Pull request merged**